src/api/kube.ts

```diff
@@ -429,10 +429,60 @@ export class KubeHelper {
   async roleBindingExist(name = '', namespace = ''): Promise<boolean> {
     const k8sRbacAuthApi = KubeHelper.KUBE_CONFIG.makeApiClient(RbacAuthorizationV1Api)
     try {
-      const { body } = await k8sRbacAuthApi.readNamespacedRoleBinding(name, namespace)
-      return this.compare(body, name)
-    } catch {
-      return false
+      await k8sRbacAuthApi.readNamespacedRoleBinding(name, namespace)
+      return true
+    } catch (e) {
+      if (e.response.statusCode === 404) {
+        return false
+      }
+
+      throw this.wrapK8sClientError(e)
+    }
+  }
+
+  async isMutatingWebhookConfigurationExists(name: string): Promise<boolean> {
```

Comment:
at milestone 1 we must guarantee that DevWorkspace Engine it not enabled on the cluster where Web Terminal is enabled.

So, apart from checking if mutating webhooks cfg exist, it would be cool to check the configured namespace inside. It's the example of Mutating Webhooks created by Web Terminal
```yaml
apiVersion: admissionregistration.k8s.io/v1
kind: MutatingWebhookConfiguration
metadata:
  labels:
    app.kubernetes.io/name: devworkspace-webhook-server
    app.kubernetes.io/part-of: devworkspace-operator
  name: controller.devfile.io
webhooks:
- admissionReviewVersions:
  - v1beta1
  clientConfig:
    ...
    service:
      name: devworkspace-webhookserver
      namespace: openshift-operators
      path: /mutate
      port: 443
  ...
  name: mutate.devworkspace-controller.svc
  namespaceSelector: {}
  objectSelector: {}
  reinvocationPolicy: Never
  rules:
  ...
- admissionReviewVersions:
  - v1beta1
  clientConfig:
    service:
      name: devworkspace-webhookserver
      namespace: openshift-operators
      path: /mutate
      port: 443
  ...
```
Pay attention on namespace: openshift-operators.

Also, about Web Terminal presence can tell Subscription:
```
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  labels:
    operators.coreos.com/web-terminal.openshift-operators: ""
  name: web-terminal
  namespace: openshift-operators
spec:
  channel: alpha
  installPlanApproval: Automatic
  name: web-terminal
  source: redhat-operators
  sourceNamespace: openshift-marketplace
  startingCSV: web-terminal.v1.1.0
```
But in general it's going to break DevWorkspaces, if there is another devworkspace controller deployed.

So, the smart logic on Chectl side could be (at least in my opinion ;) ):

1. There is no need to provide an ability to override devworkspace operator namespace, so let's just remove it.
2. Then chectl checks and fails if Web Terminal installed (Subscription exists).
3. Then chectl checks if mutatingwebhooks exists, and their namespace is different than devworkspace-operator - checlt fails and tells that WebTerminal is not uninstalled properly (if namespace if openshift-operators) or another devworkspace controller is already deployed in namespace X (it can be evaluated on chectl with webhooks object).